bombastic/walker/src/lib.rs

```diff
@@ -84,6 +84,14 @@ pub struct Run {
     /// Additional root certificates for the destination
     #[arg(long = "sender-root-certificates")]
     pub additional_root_certificates: Vec<PathBuf>,
+
+    /// Allow logging of uploaded sbom file reports.
+    #[arg(long, env, default_value_t = true, action = ArgAction::Set)]
+    pub report_enable: bool,
+
+    /// Path of the HTML output file
+    #[arg(long, default_value = "/tmp/share/report")]
+    pub report_path: Option<String>,
```

Comment:
Is there a reason the `env` attribute is missing?

Reply:
https://github.com/trustification/trustification/pull/1289#discussion_r1593742835
so I do not add it in env.

Reply:
I am not sure how this aligns with that comment.

1. There's a default value, having the type of the value `Option<T>` doesn't seem to make sense
2. It can be overridden by `--report-path` (which also seems to be used), for other args we allow overriding by `env`. I don't see any reason why we shouldn't do this in this case too.
3. The same seems true for the vexination walker.

Reply:
ok , I am not sure my understander is correct, I updated the PR, please check agian.

Reply:
You can keep the default value. Add the `env` marker and make it `String`.